helm/cfgov/templates/tests/smoke-test.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "cfgov.fullname" . }}-smoke-test-job"
+  labels: {{- include "cfgov.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+    "helm.sh/hook-delete-policy": before-hook-creation
+spec:
+  restartPolicy: Never
+  containers:
+    - name: "{{ include "cfgov.fullname" . }}-smoke-test-job"
```

Comment:
Everything here looks great, and I am going to be a bit pedantic here, but lets drop the `-job` from the name